static/docs/use-cases/data-registry.md

```diff
@@ -44,36 +36,75 @@ The advantages of using a DVC **data registry** project are:
-of our dataset, we extracted the first part into the `use-cases/cats-dogs`
-directory (illustrated below), and ran `dvc add use-cases/cats-dogs` to
-[track the entire directory](https://dvc.org/doc/command-reference/add#example-directory).
+  HTTP location). Git versioning of [DVC-files](/doc/user-guide/dvc-file-format)
+  allows us to track and audit data changes.
+
+## Building data registries
+
+Data registries are <abbr>DVC repositories</abbr>, so they can be created
+locally like any other Git + DVC <abbr>project</abbr>. However, registries
+should be available online i.e. pushed to a Git server. For example:
+
+```dvc
+$ mkdir my-data-registry && cd my-data-registry
+$ git init && dvc init
+$ git commit -am "Initialize DVC project"
+$ git remote add origin git@... # Git server URL
+$ git branch -u origin/master
+$ git push
+```
+
+What makes online data registries special, is that they mainly contain simple
+[DVC-files](/doc/user-guide/dvc-file-format) (probably no source code or
+[pipelines](/doc/command-reference/pipeline)). These [DVC-files track the
+different datasets we may want to version. The actual data will be stored in one
+or more [remote storage](/doc/command-reference/remote) locations configured in
+the <abbr>project</abbr>.
+
+A good way to organize these DVC-files is in different directories that group
+the data into separate uses, for example `images/`, `natural-language/`, etc. As
+an example, our
+[dataset-registry](https://github.com/iterative/dataset-registry) uses a
+directory for each of our website documentation sections, such as `get-started/`
+and `use-cases/`.
+
+> We use this example registry for all of our docs, where needed, for example in
+> the [Versioning](/doc/tutorials/versioning) tutorial,
+> [in Get Started](/doc/get-started/add-files), and some Command Reference
+> examples.
+
+### Adding datasets to a registry
+
+<!-- Probably we will need to touch cats & dogs example and answer (again, high level perspective) to questions like - how do we get data there, how do we update it, how do we consume it from other repos.  -->
+<!-- we should be comparing no DVC at all (ad-hoc conventions and total mess on S3) vs. the DVC Data Registry – which effectively provides some "meta" information for the same data on S3. -->
+<!-- something like - we had S3 like everyone - few buckets already (for such a small case & docs), few versions of the files in different locations, directories with images repeatings, etc - explain high level problems -->
+<!-- let’s just assume that we have a directory and we keep adding files -->
+
+Imagine a training dataset with 1000 images of cats and dogs that will be used
+to build an ML model. Without DVC, in order for a team to collaborate on this
+project, we could just uploading it to cloud storage (e.g. Amazon S3) and
+provide everyone with access.
+
+At some point though, we need to add another 1000 images to the dataset, but the
```

Comment:
We had everything in past tense (could) and now we switch to present.

Reply:
Oh this part is not to be reviewed yet, sorry for confusion. Will keep that in mind though.

Reply:
I feel present tense reads more smooth if that matters :)

Reply:
The way we're using "could" is also present tense actually

![image](https://user-images.githubusercontent.com/1477535/69835092-8c39f880-1204-11ea-834d-010541c97784.png)

But I know what you mean. It's just that some of these paragraphs use hypotetical/conditional forms and that requires these auxiliary/past verbs. Would need to rewrite the whole paragraph differently, and we definitely **could**, but I don't think this is a problem.